build/openwisp_openvpn/openvpn.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# This script will be called by cronjob to
+# update OpenVPN configurations periodically.
+source /utils.sh
+
+init_conf
+openvpn_config
+openvpn_config_checksum
+
+if [ "${OFILE}" != "${NFILE}" ]; then
+    openvpn_config_download
+    supervisorctl restart openvpn
```

Comment:
why was supervisor chosen to run OpenVPN?

Reply:
Supervisor is used because I need to restart OpenVPN after crl list is updated or configurations are changed.
If I use OpenVPN as the service keeping the docker container running, restarting would mean killing the container while supervisor helps me prevent that!

Reply:
great, can you add this explanation somewhere as a comment please?

Reply:
Done! :+1: